src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -159,14 +179,9 @@ private void handleExit() {
         GuiSettings guiSettings = new GuiSettings(primaryStage.getWidth(), primaryStage.getHeight(),
                 (int) primaryStage.getX(), (int) primaryStage.getY());
         logic.setGuiSettings(guiSettings);
-        helpWindow.hide();
         primaryStage.hide();
     }
 
-    public PersonListPanel getPersonListPanel() {
-        return personListPanel;
-    }
-
```

Comment:
May I know why you are deleting this? Is it because this method is not used?

Reply:
Yup, I originally used the function to use from HelpCommandPanel to change from the help to person list there but it was quite inefficient to get everything from there so I ended up just sending a reference to the main window itself to help which is a cyclic dependency now that I think about it, oops probably have to change stuff then.

Reply:
Ahhh, I see. We were told cyclic dependencies are bad practices in 1101 right? While its a bad practice I think it is still used haha if you take a look at w8 topics the first video for drawing class/object diagrams, they have cyclic dependency also: box and lid.
Maybe if you want to, you can create a static method in MainWindow that changes the current panel to the personlist panel then call this static method in HelpCommandPanel. Not sure if that is still cyclic dependency lol.

Reply:
Oh really? Did not know that haha
I tried using a static method but it cannot access non-static variables so that does not work either
I guess I'll try the association class then, or leave it if I cannot lolol